zkvm/src/vm.rs

```diff
@@ -29,10 +29,10 @@ pub struct TxHeader {
     /// Version of the transaction
     pub version: u64,
 
-    /// Timestamp before which tx is invalid (sec)
+    /// Timestamp before which tx is invalid (in milliseconds since the Unix epoch)
```

Comment:
I think I still prefer using seconds as less error-prone. We can bikeshed this here: https://github.com/interstellar/slingshot/issues/189

Reply:
UPD: objection withdrawn